src/components/ReportActionItem/ReportPreview.tsx

```diff
@@ -162,6 +170,28 @@ function ReportPreview({
         [chatReport?.isOwnPolicyExpenseChat, policy?.harvesting?.enabled],
     );
 
+    const confirmPayment = (type?: PaymentMethodType | undefined) => {
```

Comment:
Should we create a utility function to avoid code repetition?

Reply:
Same as above